config.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020 Chair of Applied Cryptography, Technische Universität
+// Darmstadt, Germany. All rights reserved. This file is part of
+// perun-eth-demo. Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package prnm
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/sirupsen/logrus"
+	"perun.network/go-perun/apps/payment"
+	"perun.network/go-perun/backend/ethereum/wallet"
+
+	"perun.network/go-perun/log"
+	plogrus "perun.network/go-perun/log/logrus"
+)
+
+// Config complete configuration needed to operate the Client.
+type Config struct {
+	Alias string
+	// Address will be unlocked in the KeyStore with password
+	KeyStorePath, Password string
+	address                *Address
+	// used for persistence
+	DatabasePath string
+	ETHNodeURL   string
+	IP           string
+	Port         uint16
+	// TODO loglevel
+}
+
+// Deployed on ropsen
+var assetAddr = common.HexToAddress("0xC2F95e626123a61beD88752475b870eFC4a5F453")
+var adjudicatorAdr = common.HexToAddress("0x94503E14E26A433C0802E04f2aC1Bb1CE77321F5")
+
+// Random address
+var appDef = common.HexToAddress("0x0583849a3C5F37aEfAb8cCcA303f9229AdF5A32a")
+
+func NewConfig(alias, keyStorePath, password string, address *Address, databasePath, ETHNodeURL, ip string, port int) *Config {
+	return &Config{alias, keyStorePath, password, address, databasePath, ETHNodeURL, ip, uint16(port)}
+}
+
+func init() {
+	payment.SetAppDef((*wallet.Address)(&appDef))
+	logger := logrus.New()
+	logger.SetLevel(logrus.TraceLevel)
```

Comment:
I'd set it to `Info` and give the `prnm` package a setter `SetLogLevel(int)` (what type is `logrus.Level`? Does gomobile support enums?)

Reply:
Initially we wanted to put it in the `Config` but then the `Client` would have to set it so i agree and will create a `SetLogLevel(int)` function.  
Using `logrus.Level` seems not to work with *go-mobile* despite being an `uint32`.